### PR TITLE
Test: modify check-docker

### PR DESCRIPTION
**Pull request by dperpeet**
use busybox instead of building a docker image from scratch

### CODE REVIEW

File: test/testsuite-prepare [hunk 1]
```diff
@@ -65,4 +65,4 @@ fi
 ./vm-reset
 ./vm-install -f cockpit $rpms
 
-fakeroot ./make-docker-image container-probe.tar.gz container-probe
+fakeroot ./make-docker-image container-probe
```
Comment (mvollmer) [inline test/testsuite-prepare]: We don't need fakeroot anymore.

Comment (dperpeet) [inline test/testsuite-prepare]: thanks, already removed that in a non-pushed commit

**Comment by mvollmer** [comment]
The container-probe executable itself seems to be problematic.  Maybe because it is statically linked?

Can we reimplement its functionality with pure busybox?  shell script, whatever.

**Comment by dperpeet** [comment]
rebased and updated

**Comment by stefwalter** [comment]
check-docker fails like this:

```
  File "./check-docker", line 294, in <module>
    if os.environ['TEST_OS'] and os.environ['TEST_OS'] == 'fedora-22':
  File "/usr/lib64/python2.7/UserDict.py", line 23, in __getitem__
    raise KeyError(key)
KeyError: 'TEST_OS'
```

**Comment by stefwalter** [comment]
@mvollmer In addition Fedora contains docker-1.6.0-0.1.rc5.fc22.x86_64

Can the entire hack be removed?

**Comment by dperpeet** [comment]
> Can the entire hack be removed?

discussion for a separate pr, I think

**Comment by dperpeet** [comment]
> check-docker fails like this:

I made this more robust and also added the environment variables `TEST_OS` and `TEST_ARCH` to `check-verify`, with defaults if they aren't set: https://github.com/cockpit-project/cockpit/pull/1985/files#diff-f1702748bd7ddb4991197553bb882f67R24

**Comment by stefwalter** [comment]
check-docker failed

```
OK
Wrote check-docker-testBasic-FAIL.png
Journal database copied to check-docker-testBasic-10.111.111.14-FAIL.journal
EWarning: Permanently added '10.111.111.14' (ECDSA) to the list of known hosts.

======================================================================
ERROR: testBasic (__main__.TestDocker)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./check-docker", line 75, in testBasic
    b.enter_page("containers")
  File "/home/hubbot/hubbot/30e08e8ab2f77b0cf8687e81842b67816086e190/test/testlib.py", line 299, in enter_page
    self.wait_visible('#' + id)
  File "/home/hubbot/hubbot/30e08e8ab2f77b0cf8687e81842b67816086e190/test/testlib.py", line 218, in wait_visible
    return self.wait_js_func('ph_is_visible', selector)
  File "/home/hubbot/hubbot/30e08e8ab2f77b0cf8687e81842b67816086e190/test/testlib.py", line 209, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))), timeout=self.phantom_wait_timeout)
  File "/home/hubbot/hubbot/30e08e8ab2f77b0cf8687e81842b67816086e190/test/testlib.py", line 547, in wait
    return self.run({'cmd': 'wait', 'cond': cond, 'timeout': timeout*1000})
  File "/home/hubbot/hubbot/30e08e8ab2f77b0cf8687e81842b67816086e190/test/testlib.py", line 513, in run
    raise Error("timeout")
Error: timeout
```

**Comment by dperpeet** [comment]
> check-docker failed

Failed on F22 with enabled test -> hack is still necessary, disabling again.

**Pull request closed without merging**